common/src/main/java/org/dash/wallet/common/data/CurrencyInfo.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2019 Dash Core Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.dash.wallet.common.data;
+
+import android.annotation.TargetApi;
+import android.os.Build;
+
+import java.util.Currency;
+import java.util.HashMap;
+
+/**
+    @author: Eric Britten
+ */
+
+public class CurrencyInfo {
+
+    /*
+        Older android devices will have obsolete currency codes
+        that do not match what the prices sources currently used.
+
+        e.g. in Belarus (BY), the currency changed from BYR to BYN
+        at the end of 2016.  Older phones have BYR.
+
+        This app will read BYR from the device and then query BYN
+        in the exchange rates list, but must get the name of BYR
+        from the device.
+     */
+    private static HashMap<String, String> obsoleteCurrencyMap;
+
+    /*
+        These currencies are listed in the price data, but are not
+        ISO 4217 currency codes.  This will map those codes to the
+        currency names.
+     */
+    private static HashMap<String, String> otherCurrencyMap;
+
+    /*
+        These currencies are listed in the price data and have the
+        same name as a different currency.  This differs from
+        obsoleteCurrencyMap because both of these codes exist in the
```

Comment:
Should we put **USDC** here as well?

Reply:
It's in the [bitpay response](https://bitpay.com/rates)

Reply:
Yes, good idea.

Reply:
this was added

Reply:
There are other currencies to add text for that come from BtcPay, so I will add those also.